web/src/datastore.js

```diff
@@ -99,9 +96,9 @@ const connectFeature =
 	firestoreConnect((store, props) => [
 		{
 			collection: `projects/${store.match.params.projectId}/records`,
-      where: [
-         ['featureId', '==', store.match.params.featureId]
-      ],
+			where: [
+				['featureId', '==', store.match.params.featureId]
+			],
```

Comment:
These are tabs. Please use spaces instead.

Reply:
Gino, if I open any of your files and replace indent with two spaces than I see a diff. I am really confused with what you are asking me for here because I literally copy pasted what is in lines that you consider proper...

Reply:
If you highlight them in the diff above you'll see they're actually tabs; I believe your editor may be replacing the two spaces with tabs. Is that possible?

Reply:
Ah, I see what you mean, I may have misunderstood, since when I highlight other text in GitHub it also gives the same effect. Please disregard, I'll figure out what's goin gon in a follow-up!